src/modules/App.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable */
+import { getData, countOccurrences } from './ApiSimpson.js';
+import iconLike from '../assests/like.png';
+import getLikes from './GetLikes.js';
+import sendLike from './SendLike.js';
+import addComment from './addComments.js';
+import { API_URL1 } from './Vars.js';
+import getComments from './GetComments.js';
+
+// Render cards
+async function renderCards() {
+  // Data from simpson api
+  const data = await getData(API_URL1);
+  const CardsContainer = document.getElementById('dinamic-content');
+
+  const nLikesArray = await Promise.all(
+    data.map((episode) => getLikes(episode.id)),
+  );
+  data.forEach((episode, index) => {
+    const nLikes = nLikesArray[index];
+
+    // data.forEach((episode) => {
+    // for (const episode of data) {
+    // const nLikes = await getLikes(episode.id);
```

Comment:
- [ ] Same here,  Please, this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
well resolved! ✔️